draft-sharabayko-mops-srt.md

```diff
@@ -1698,9 +1698,9 @@ for sending on the SRT sender side. Basically, the sender timestamp in the recei
 is adjusted to the receiver’s local time (compensating for the time drift or different 
 time zones) before releasing the packet to the application. Packets can be withheld by 
 the SRT receiver for a configured receiver delay. A higher delay can accommodate a larger 
-uniform packet drop rate, or a larger packet burst drop. Packets received after their 
-"play time" are dropped if the Too-Late Packet Drop feature is enabled 
-({{too-late-packet-drop}}).
+uniform packet drop rate, or a larger packet burst drop. For example, in the case of video
+streaming with the TSBPD mechanism, packets received after their "play time" are dropped
+if the Too-Late Packet Drop feature is enabled ({{too-late-packet-drop}}).
```

Comment:
> Packets received after their "play time" are dropped if the Too-Late Packet Drop feature is enabled 

The initial phrasing was not an example, but the description of the dropping condition.

**Maybe something like this?**

Packets received after their "play time" are dropped if the Too-Late Packet Drop feature is enabled ({{too-late-packet-drop}}).
For example, in the case of live video streaming TSBPD and TL Packet Drop mechanisms allow to intentionally drop those packets that were lost and have no chance to be retransmitted before their play time. Thus providing a known and a fixed end-to-end latency of the stream.

Reply:
Since I don't think 'live mode' is only for video streaming, I was unsure if "play time" is the right term. For example, in theory, we can send time-series sensor data in live mode of SRT so I thought we might need a more general term to replace "play time". 

However,  I think the additional explanation you have attached is also a good idea.